applications/ShapeOptimizationApplication/custom_utilities/mapping/mapper_vertex_morphing_symmetric.cpp

```diff
@@ -218,6 +221,7 @@ void MapperVertexMorphingSymmetric::ComputeMappingMatrix()
 
     // fixed size vec
     NodeVectorType neighbor_nodes( max_number_of_neighbors );
+    #pragma omp for
```

Comment:
this should use the parallelUtils with ThreadLocalStorage I strongly recommend (https://github.com/KratosMultiphysics/Kratos/wiki/Parallel-Utilities#basic-statically-scheduled-for-loop)

However IIRC there is a bit more to do, e.g. making sure inserting in the MappingMatrix should be atomic etc

Reply:
ah ok

well if you know this then I would not add the atomic, but a comment somewhere

If you can write the algorithm without need for synchronization is better IMO as faster

I took a look at the file, actually it seems to be ok as is!

I.e. if you want you could even merge as is. I recommend using the parallel utils but is is up to you